File: Code_ground/NO_0113/113.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 24                   | #
# | 1비트 개수 세기            | #
# ---------------------------- #


def run():
    T = int(r_input())

    for testcase in range(1, T + 1):
        N = int(r_input())

        print('Case #' + str(testcase))
        print(bin(N).count('1'))


if __name__ == "__main__":
    run()

File: Code_ground/NO_0113/113.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 28                   | 
 | 1비트 개수 세기            | 
 ---------------------------- 
*/

int main(){
    cin.tie(NULL);
    cout.tie(NULL);
    ios_base::sync_with_stdio(false);
    int T;

    cin >> T;
    

    for(int testcase = 1; testcase <= T; testcase ++){
        int N, res = 0;
        
        cin >> N;

        for(int mul = 29; mul >= 0; mul --) res += N >> mul & 1;

        cout << "Case #" << testcase << "\n";
        cout << res << endl;
    }
    return 0;
}
